modules/ROOT/pages/dw-coercions-types.adoc

```diff
@@ -1,5 +1,6 @@
 = Coercions Types (dw::util::Coercions)
 
+[%header, cols="1,2a,3a"]
```

Comment:
TODO (DEV): [%header, cols="1,2a,3a"] 

add to table output for types, annotations, and variables(?) docs